Cargo.toml

```diff
@@ -55,3 +56,10 @@ schemars = "0.8.21"
 dashmap = "6.0.1"
 uuid = "1"
 chrono = "0.4"
+
+# sea-orm and sqlx's [libsqlite3-sys](https://github.com/rusqlite/rusqlite/tree/master/libsqlite3-sys) version conflict with openDAL
```

Comment:
I think it's preferable to disable OpenDAL's SQLite support for the time being rather than adding `patch.crates-io` here.

Reply:
@Xuanwo 老哥，你们是基于opendal在做databend的创业项目吗

Reply:
> @Xuanwo 老哥，你们是基于opendal在做databend的创业项目吗

Databend **uses** OpenDAL but is not **based on** it. OpenDAL is merely a small component used by Databend.

Reply:
respect

Reply:
The OLAP market is now very competitive

Reply:
> I think it's preferable to disable OpenDAL's SQLite support for the time being rather than adding `patch.crates-io` here.

I had tried to modify `spring-opendal` dep config, but there seems no way to avoid this

can't find documents, but it seems https://github.com/rust-lang/cargo/issues/10801 mentioned, `cargo.lock` has all dep, and that causes compile conflict

Reply:
> can't find documents, but it seems [rust-lang/cargo#10801](https://github.com/rust-lang/cargo/issues/10801) mentioned, `cargo.lock` has all dep, and that causes compile conflict

Just remove `services-sqlite`?

Reply:
> > can't find documents, but it seems [rust-lang/cargo#10801](https://github.com/rust-lang/cargo/issues/10801) mentioned, `cargo.lock` has all dep, and that causes compile conflict
> 
> Just remove `services-sqlite`?

thanks, it works